desktop/cmp/persistenceManager/impl/SaveDialogModel.ts

```diff
@@ -0,0 +1,86 @@
+import {FormModel} from '@xh/hoist/cmp/form';
+import {HoistModel, managed, PlainObject, TaskObserver, XH} from '@xh/hoist/core';
+import {lengthIs, required} from '@xh/hoist/data';
+import {makeObservable} from '@xh/hoist/mobx';
+import {PersistenceManagerModel} from '../PersistenceManagerModel';
+
+export class SaveDialogModel extends HoistModel {
+    parentModel: PersistenceManagerModel;
+
+    @managed saveTask = TaskObserver.trackLast();
```

Comment:
`TaskObserver` does not have a `destroy()` method, so no need to mark `managed`. However, can make `readonly`